app/src/marketplace/components/StreamPreviewPage/streamPreviewPage.pcss

```diff
@@ -48,7 +48,7 @@
   .closeButton {
     padding: 0 10px;
     cursor: pointer;
-    background: none;
+    background: none !important;
```

Comment:
The one on the stream preview page was more of a button styling issue from before the fixes. Also fixed.